handbook/engineering/hiring/software-engineer-cloud-frontend.md

```diff
@@ -0,0 +1,98 @@
+# Software Engineer - Cloud - Frontend
+
+The [Cloud team](../cloud/index.md) is looking for an experienced frontend engineer who knows how to build intuitive user experiences to make the power of [Universal Code Search](https://about.sourcegraph.com/product) accessible to everyone on [sourcegraph.com](https://sourcegraph.com/). You will have a lot of ownership to solve tough technical and UX problems that will drive massive adoption of our product by developers all around the world.
```

Comment:
@tsenart If I recall your requirement was for a fairly senior frontend engineer. If that is the case I think we should call that out specifically.

Reply:
Do you mean change "experienced" to "senior"?

Reply:
Changed to senior, thanks.